test/docker-compose_nms.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.4'
+
```

Comment:
Can we re-name this file as "docker-compose?"

Reply:
If this is a compose file for the nms rather than the catalog, calling it as such seems appropriate to me

Reply:
It has nothing to do with testing the catalog service. It's just for starting the NMS service for CI/local testing to run unit tests. It makes more sense to keep its name as is.

Reply:
I'm fine with `-f <file>` in this case but I don't feel that strongly about it